csharp/test/Ice/enums/AllTests.cs

```diff
@@ -65,34 +65,6 @@ public static Test.ITestIntfPrx allTests(global::Test.TestHelper helper)
 
             output.WriteLine("ok");
 
```

Comment:
We no longer test enum streaming?

Reply:
we test that with Ice/stream test, I think this removed test was only because 1.0 has variable length enums, and we checked the size.